Flow.Launcher.Plugin/Flow.Launcher.Plugin.csproj

```diff
@@ -42,6 +42,7 @@
     <WarningLevel>4</WarningLevel>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <Prefer32Bit>false</Prefer32Bit>
+    <DocumentationFile>..\Output\Debug\Flow.Launcher.Plugin.xml</DocumentationFile>
```

Comment:
This option is redundant, isn't it? You have only added it to the Debug group, currently, but the CI-built dll file has all the comments...

Reply:
I add it for generating the api docs (which I hasn't published, and directly reference this project). It doesn't need to present in release.